app/registries/foreman/settings/provisioning.rb

```diff
@@ -131,7 +131,7 @@
     setting('ct_arguments',
       type: :array,
       description: N_("CoreOS transpiler (ct) arguments as an comma-separated array"),
-      default: ['--pretty', '--files-dir', Rails.root.join('config', 'ct').to_s],
+      default: Rails.env.test? ? [] : ['--pretty', '--files-dir', Rails.root.join('config', 'ct').to_s],
```

Comment:
This is already done in `test/fixtures/settings.yml`, why is it required here?

Reply:
IDK, but without my changes the snapshot task doesn't work.

Reply:
We probably don't load the fixtures for these tests, I'll need to investigate